tests/phpunit/tests/theme/wpTheme.php

```diff
@@ -375,6 +375,33 @@ public function test_get_files_nonexistent_theme() {
 		$this->assertEmpty( $files );
 	}
 
+	/**
+	 * Test wp_customize_url with existing query args.
+	 *
+	 * @ticket 63632
+	 */
+	public function test_wp_customize_url_with_existing_query_args() {
+		$clean_admin_url = admin_url( 'customize.php' );
+
+		// Ensure the existing query arg is present in the URL.
+		add_filter(
+			'admin_url',
+			static function ( $url ) {
+				return add_query_arg( 'existing_arg', 'value', $url );
+			}
+		);
+		$this->assertSame( esc_url( $clean_admin_url . '?existing_arg=value&theme=theme' ), wp_customize_url( 'theme' ) );
+
+		// Ensure the theme query arg is replaced with the new value.
+		add_filter(
+			'admin_url',
+			static function ( $url ) {
+				return add_query_arg( 'theme', 'value', $url );
+			}
+		);
+		$this->assertSame( esc_url( $clean_admin_url . '?existing_arg=value&theme=theme' ), wp_customize_url( 'theme' ) );
+	}
```

Comment:
How about adding another test case for when the `$stylesheet` arg is absent entirely? Granted, the modified code wouldn't be invoked at all, but that's worth testing as well.

Reply:
I added this test too. I considered some special chars other that a space. Take a look and see if there might be others we should look out for. I don't think many will be used, but we can have a little more certainty.